src/app/components/cds/alerts-and-prompts/Alert.js

```diff
@@ -1,167 +1,110 @@
+// DESIGNS: https://www.figma.com/file/7ZlvdotCAzeIQcbIKxOB65/Components?type=design&node-id=4-45716&mode=design&t=G3fBIdgR6AWtOlNu-4
```

Comment:
@dariusk and @amoedoamorim not sure if this was the style of comment you were imaging, but if not, I am happy to change it to whatever is best for the dev teams

Reply:
:LGTM:

Reply:
yeah, works for me!